finder-app/finder.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# AESD Assignment 1 
+# Author: Gautama Gandhi
+# Description: This file is the finder.sh
+
+filesdir=$1
+searchstr=$2
+
+#Check number of arguments
+
+if [ $# -ne 2 ]
+then
+    echo "Incorrect number of parameters. Expected 2!"
```

Comment:
@GautamaGandhi how could you make the error message more descriptive to help the user through these error cases?